packages/artillery/lib/platform/aws/aws-cloudwatch.js

```diff
@@ -47,16 +47,36 @@ function setCloudwatchRetention(
         );
         clearInterval(interval);
       } catch (error) {
-        if (error & (error.code != 'ResourceNotFoundException')) {
-          console.log('WARNING: Unexpected error setting retention policy:');
-          console.log(error);
+        const resumeTestMessage =
+          'The test will resume without setting the retention policy.';
```

Comment:
```suggestion
          'The test will continue without setting the retention policy.';
```

"Resume" implies something was paused imo which isn't the case